src/test/resources/validationTest/breakendTests/nonreciprocalBalancedWithLargeWiggle/test_report.json

```diff
@@ -0,0 +1 @@
:0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":50,"upper":99,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":100,"upper":199,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":200,"upper":399,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":400,"upper":799,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":800,"upper":1599,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":1600,"upper":3199,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":3200,"upper":6399,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":6400,"upper":12799,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":12800,"upper":25599,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":25600,"upper":51199,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":51200,"upper":102399,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":102400,"upper":500000,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":500001,"upper":1000000,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0},{"T":0,"lower":1000001,"upper":-1,"empty":true,"f1":"NaN","fp":0,"t":0,"fn":0,"tp":0,"tn":0,"spc":"NaN","fdr":"NaN","ppv":"NaN","tpr":"NaN","_TP":0,"_FP":0,"_TN":0,"_FN":0}],"sum_count":{"T":4,"lower":-1,"upper":-1,"empty":false,"f1":1.0,"fp":0,"t":4,"fn":0,"tp":4,"tn":0,"spc":"NaN","fdr":0.0,"ppv":1.0,"tpr":1.0,"_TP":4,"_FP":0,"_TN":0,"_FN":0},"sum_per_base_count":{"T":14,"lower":-1,"upper":-1,"empty":false,"f1":1.0,"fp":0,"t":14,"fn":0,"tp":14,"tn":0,"spc":"NaN","fdr":0.0,"ppv":1.0,"tpr":1.0,"_TP":14,"_FP":0,"_TN":0,"_FN":0},"bin_breaks":[1,2,3,4,5,6,7,8,9,10,19,29,39,49,99,199,399,799,1599,3199,6399,12799,25599,51199,102399,500000,1000000]}}}
```

Comment:
Instead of Translocation_Duplication in the JSON, might be better to use TranslocationDup or TranslocationDel so that it renders without overlapping in the HTML report.